lib/ruby-stackoverflow/client/user_helper.rb

```diff
@@ -114,6 +114,11 @@ def users_top_questions_with_given_tags(id, tags, options = {})
         user_response(options, url)
       end
 
+      def user_top_tags(id, options ={})
+        url = id + '/top-tags'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.